src/SFA.DAS.EAS.Portal.Client/Configuration/RecruitApiClientConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using SFA.DAS.Http;
+
+namespace SFA.DAS.EAS.Portal.Client.Configuration
+{
+    public class RecruitApiClientConfiguration : IAzureADClientConfiguration
+    {
+        public string ApiBaseUrl { get; set; }
+        public string TimeoutTimeSpan { get; set; }
```

Comment:
Is it worth this being a timeout in seconds which we convert to a timespan in the code?

Reply:
The advantage of using a string that is fed to `TimeSpan.Parse`, is that it allows us to use any value that `TimeSpan` supports. E.g. we could set the timeout to 2.5 seconds.

Reply:
I've also introduced a rexeg in the config's json schema, to check any supplied value for correctness, to address your spoken concern of incorrectly formatted values not being picked up in the config system.